src/Rules/Library/ControlShouldSupportTextPatternEditWinform.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Axe.Windows.Core.Bases;
+using Axe.Windows.Core.Enums;
+using Axe.Windows.Rules.PropertyConditions;
+using Axe.Windows.Rules.Resources;
+using System;
+using static Axe.Windows.Rules.PropertyConditions.ControlType;
+using static Axe.Windows.Rules.PropertyConditions.Framework;
+
+namespace Axe.Windows.Rules.Library
+{
+    [RuleInfo(ID = RuleId.ControlShouldSupportTextPatternEditWinform)]
+    class ControlShouldSupportTextPatternEditWinform : Rule
+    {
+        public ControlShouldSupportTextPatternEditWinform()
+        {
+            this.Info.Description = Descriptions.ControlShouldSupportTextPattern;
+            this.Info.HowToFix = HowToFix.ControlShouldSupportTextPattern;
```

Comment:
Does it make sense to use the same values from `ControlShouldSupportTextPattern` or would you recommend new values for this rule?

Reply:
We've been using the same description and HowToFix for these. When we make the AIWin changes, if both HowToFix and FrameworkIssueLink are set, we'll only display the FrameworkIssueLink. That provides us with downlevel compatibility if an older client opens a newer a11ytest file.